src/components/ApplnBar.js

```diff
@@ -73,6 +90,8 @@ const ApplnBar = () => {
     >
       <List>
         {[
+          'USER',
```

Comment:
Let's move this to the end of the list as it won't be the most commonly used feature.